spark/spark-version/1.5-plus/src/main/scala/org/apache/spark/ml/MlTransformer.scala

```diff
@@ -23,7 +23,7 @@ import org.apache.spark.sql.types.StructType
  * Extends MlTransformer and override process to gain compatibility with
  * both spark 1.5 and spark 2.0.
  */
-abstract class MlTransformer extends Transformer{
+abstract class MlTransformer extends Model[MlTransformer]{
```

Comment:
I think we should have either MlTransformer & DlTransformer, or MLTransformer & DLTransformer